docs/content/configuration/creating_pvc.md

```diff
@@ -4,9 +4,11 @@ Create a PersistentVolumeClaim (PVC) YAML file for a persistent volume (PV).
 
 The IBM® block storage CSI driver supports using both file system and raw block volume modes.
 
-**Important:** If not defined, the default mode is `Filesystem`. Be sure to define the mode as `Block` if this configuration is preferred.
+**Important:**
+  - If not defined, the default mode is `Filesystem`. Be sure to define the mode as `Block` if this configuration is preferred.
+  - Only use the `volumegroup` label if the PVC is not using a StorageClass with a defined `volume_group`. The PVC `volumegroup` label value is the key defined in the VolumeGroup YAML. The labels are not pre-defined. Be sure to match the selector in the target volume group (`spec.source.selector`). For an example of creating a PVC using the VolumeGroup configuration, see [Creating a PVC in a volume group](#creating-a-pvc-in-a-volume-group).
```

Comment:
the name of label does not have to be 'volumegroup'.
it only has be the same as defined  in the key selector in the volumegroup yaml.
both key and value must match for the pvc to be added/created in the volume group.